site/usage/applications.md

```diff
@@ -59,3 +59,4 @@ We mark featured plugins and tools with a <span class="octicon octicon-star"></s
 -   [marked-vega](https://www.webcomponents.org/element/PolymerVis/marked-vega) is a Polymer web component to parse image/code markdowns into Vega and Vega-Lite charts.
 -   [gulp-marked-vega](https://github.com/e2fyi/gulp-marked-vega) is a gulp plugin (comes with a cli tool also) to replace [marked-vega](https://www.webcomponents.org/element/PolymerVis/marked-vega) markdown syntax with base64 embedded image tags, so that any standard markdown parser can render the Vega and Vega-Lite charts without modifying their render rules.
 -   [idyll-vega-lite](https://github.com/idyll-lang/idyll-vega-lite) is a component that allows you to embed Vega-Lite graphics inside of [Idyll markup](https://idyll-lang.org), an interactive markup language.
+-   [generator-veeg](https://github.com/millette/generator-veeg) is a vega and vega-lite boilerplate generator for [Yeoman](http://yeoman.io/).
```

Comment:
Please capitalize Vega and Vega-Lite.

Reply:
Sure thing. I also fixed the generator readme, thanks for the tip!